src/components/transaction/SolverCompetition/ClearingPrices.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from 'react'
+import { useNetworkId } from 'state/network'
+import { EmptyItemWrapper } from 'components/common/StyledUserDetailsTable'
+import { PricesCard } from 'components/transaction/SolverCompetition/styled'
+import { formatSmart, safeTokenName, TokenErc20 } from '@gnosis.pm/dex-js'
+import { Network } from 'types'
+import TokenImg from 'components/common/TokenImg'
+import { HIGH_PRECISION_SMALL_LIMIT, NO_ADJUSTMENT_NEEDED_PRECISION } from 'apps/explorer/const'
+import { getImageAddress } from 'utils'
+import { invertPrice } from '@gnosis.pm/dex-js/build-esm/utils/price'
+import Icon from 'components/Icon'
+import { faExchangeAlt } from '@fortawesome/free-solid-svg-icons'
+import BigNumber from 'bignumber.js'
+import { NATIVE_TOKEN_PER_NETWORK, TEN_BIG_NUMBER } from 'const'
+import { Order } from 'api/operator'
+import { AuctionPrices } from '@cowprotocol/cow-sdk'
+
+type Props = {
+  orders: Order[]
+  prices: AuctionPrices
+}
+
+type ItemProps = {
+  token: TokenErc20
+  amount: BigNumber
+  network: Network
+}
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+const Item: React.FC<ItemProps> = (props) => {
+  const { token, network, amount } = props
+
+  const [invertedPrice, setInvertedPrice] = useState<boolean>(false)
+  console.log(token, amount.toNumber())
+  const calculatedPrice = amount.div(TEN_BIG_NUMBER.exponentiatedBy(36 - token.decimals.valueOf()))
```

Comment:
What is 36? Would be great to move it to a const with an appropriate name + comment